hw/top_earlgrey/ip/ast/rtl/ast.sv

```
@@ -833,7 +833,7 @@ ast_reg_top u_reg (
   .reg2hw ( reg2hw ),
   .hw2reg ( hw2reg ),
   .intg_err_o ( intg_err ),
-  .devmode_i ( 1'b0 )
+  .devmode_i ( 1'b1 )
```

@tjaychen @arnonsha @Jacob-Levy FYI. 
With this set to 0 we get many errors in the `chip_tl_errors` test since the CSR node does not error back.

---

I see that all other modules indeed have devmode=1, which means they report errors when illegal transaction occurs. From security point of view, isn't this what we're looking for ? I don't see the value of devmode = 0, can you explain why when masked errors can/should be used ?

---

@msfschaffner The value or `.devmode_i ( 1'b0 )` was given by Google in the past ro the AST... what changed?
Should the change apply to the `close source` as well?

---

Thanks @eunchan
This makes sense, however all modules that I can see are connected to 1'b1, including keymgr, kmac, lc and otbn.

---

That makes sense for the 1st gen. Thanks @tjaychen

---

@eunchan Thanks for explaining!
@msfschaffner  is asking to set `devmoe=1` that is why I asked "what changed"..